ReleaseNotes-2.9

```diff
@@ -61,6 +61,7 @@ Shipment:
 Bugfixes:
  * Deletion of groups with users was not working
  * Fixed generation of the breadcrumb for the Image Editor (Kiwi)
+ * Using ActiveSupport::JSON as encoder everywhere prevent to fail on Notification creation with events that have a very big payload
```

Comment:
Nitpick: Seems to me like this should be squashed into 3af96f73a391594c8435db97fa9f51ef67217df6